frontend/src/pages/connectionTypes/manage/ManageConnectionTypePage.tsx

```diff
@@ -166,7 +166,7 @@ const ManageConnectionTypePage: React.FC<Props> = ({ prefill, isEdit, onSave })
               </FormGroup>
               <FormGroup label="Enable" fieldId="connection-type-enable">
                 <Checkbox
-                  label="Enable users in your organization to use this connection type when adding connections."
+                  label="Connection is enabled and therefore available to use by users in your org"
```

Comment:
Is this correct? The mocks have the original text.

Reply:
The `Create` page had this text. I commented in the Jira and they updated the create page to match the original text. I will revert this.